src/main.rs

```diff
@@ -1,18 +1,42 @@
-use std::env;
 use tokio;
 
 mod nubank;
 
+mod simple_user_input {
+    use std::io;
+    pub fn get_input(prompt: &str) -> String {
+        println!("{}", prompt);
+        let mut input = String::new();
+        match io::stdin().read_line(&mut input) {
+            Ok(_goes_into_input_above) => {}
+            Err(_no_updates_is_fine) => {}
+        }
+        input.trim().to_string()
+    }
+}
+use simple_user_input::get_input;
+
 #[tokio::main]
 async fn main() {
-    let (cert_path, cpf, password) = match (
-        env::var("CERT_PATH"),
-        env::var("CPF"),
-        env::var("NUBANK_PASSWORD"),
-    ) {
-        (Ok(cert_path), Ok(cpf), Ok(password)) => (cert_path, cpf, password),
-        _ => panic!("Missing NUBANK_PASSWORD env variable!!!"),
-    };
+    let cpf = get_input("CPF: ");
+    let password = get_input("Nubank password: ");
```

Comment:
Why move from env variables into manual input? I think that env variables and flag parsing are a better solution

Reply:
It's only a way of "test the code" in some sense. It will be removed in the future.